xen/include/xen/iommu.h

```diff
@@ -195,6 +206,12 @@ struct iommu_ops {
     int __must_check (*iotlb_flush_all)(struct domain *d);
     int (*get_reserved_device_memory)(iommu_grdm_t *, void *);
     void (*dump_p2m_table)(struct domain *d);
+#ifdef CONFIG_HAS_COPROC
+    int (*assign_coproc)(struct domain *d, device_t *dev);
```

Comment:
I think iommu_ prefix would be redundant here because these are iommu_ops and nothing else.

Look, no one of the existing ops has iommu_ prefix.
 
static const struct iommu_ops ipmmu_vmsa_iommu_ops = {
	.init = ipmmu_vmsa_domain_init,
	.hwdom_init = ipmmu_vmsa_hwdom_init,
	.alloc_page_table = ipmmu_vmsa_alloc_page_table,
	.free_page_table = ipmmu_vmsa_free_page_table,
	.teardown = ipmmu_vmsa_domain_teardown,
	.iotlb_flush = ipmmu_vmsa_iotlb_flush,
	.assign_device = ipmmu_vmsa_assign_dev,
	.reassign_device = ipmmu_vmsa_reassign_dev,
	.map_pages = ipmmu_vmsa_map_pages,
	.unmap_pages = ipmmu_vmsa_unmap_pages,
	.dump_p2m_table = ipmmu_vmsa_dump_p2m_table,
#ifdef CONFIG_HAS_COPROC
	.assign_coproc = ipmmu_vmsa_assign_coproc,
	.deassign_coproc = ipmmu_vmsa_deassign_coproc,
	.disable_coproc = ipmmu_vmsa_disable_coproc,
	.enable_coproc = ipmmu_vmsa_enable_coproc,
#endif
};